com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -83,6 +83,7 @@ public void HandleConnectionApproved(ulong clientId, Stream stream, float receiv
 
                 NetworkManager.ConnectedClients.Add(NetworkManager.LocalClientId, new NetworkClient { ClientId = NetworkManager.LocalClientId });
 
+                m_NetworkManager.NetworkMetrics.TrackConnection(NetworkManager.LocalClientId);
```

Comment:
What's the advantage of this as opposed to fetching the connection ID in the network manager on every metrics call?

Reply:
I just realized I forgot to reuse it elsewhere, but I was under the impression at the moment that we could reuse the data instead of reallocation. But let me change this in a moment.